migration/testdata/v1_JobLogger.post

```diff
@@ -0,0 +1,3 @@
+/ffs/joblogger_v2/ID1/QmPewMLNZEgnLxaenjo9Q5qwQwW3zHZ7Ac973UmeJ6VWHE/1602952162298722885,{"Cid":{"/":"QmPewMLNZEgnLxaenjo9Q5qwQwW3zHZ7Ac973UmeJ6VWHE"},"RetrievalID":"","Timestamp":1602952162298722885,"Jid":"ad6da2a0-5465-4275-a330-2537062765c8","Msg":"Deal 763168 with miner f022142 is active on-chain"}
```

Comment:
Here the pre and post files for each test case.
A new datastore is populated with `.pre` file key-values, the migration is run, and then it's asserted if the datastore is exactly in the `.post` key-values state.